test/cmdlineTests/exp_base_literal/output

```diff
@@ -274,8 +274,16 @@ object "C_80" {
 
             }
 
-            function panic_error() {
-                invalid()
+            function panic_error_0x11() {
+                mstore(0, 35408467139433450592217433187231851964531694900788300625387963629091585785856)
```

Comment:
Isn't this supposed to be first 4 bytes of `keccak256("Panic(uint256)")` followed by `28` bytes of zero padding?

Reply:
Can we avoid writing it explicitly here?

Reply:
It's the job of the optimizer to find a better representation for numbers, and we actually already have a routine, but it is only implemented for the old codegen optimizer